src/views/register/index.vue

```diff
@@ -227,45 +289,41 @@ export default {
       template: '',
       templateGlobals: {},
 
-      checks: [
-        {
-          title: 'flows',
-          checked: false,
-          loading: false,
-        },
-        {
-          title: 'AI',
-          checked: false,
-          loading: false,
-        },
-        {
-          title: 'whatsapp_demo',
-          checked: false,
-          loading: false,
-        },
-      ],
+      checks: [],
+
+      organizationFormIsValid: false,
+      projectFormIsValid: false,
+      templateFormIsValid: false,
     };
   },
 
+  created() {
+    if (this.isCreatingOrgView) {
+      this.page = 'organization';
+    } else if (this.isCreatingProjectView) {
+      this.page = 'project';
+    }
+  },
+
   methods: {
-    ...mapActions(['updateProfile', 'addInitialInfo', 'createOrg']),
+    ...mapActions(['updateProfile', 'addInitialInfo']),
 
     filter,
 
-    nextPage() {
-      const pageIndex = this.pages.indexOf(this.page);
+    lowerCaseButFirst(sentence) {
```

Comment:
a suggestion of other possible names:

- `capitalizeFirst`
- `firstUpperRestLower`
- `capitalizeFirstLowercaseRest`

Reply:
I changed it to `lowerCaseExceptTheFirstLetter`, because this function isn't capitalizing the first letter, which could cause a misunderstanding.